packages/algob/test/mocks/tx.ts

```diff
@@ -6,7 +6,7 @@ import algosdk, {
 	SuggestedParams,
 } from "algosdk";
 
-import { ConfirmedTxInfo } from "../../src/types";
+import { ConfirmedTxInfo, TxnReceipt } from "../../src/types";
```

Comment:
do we need the ConfirmedTxInfo import if we dont use it anymore?

Reply:
some methods do use it

Reply:
I will check if we can remove those as well